src/Pages/BeerGoggles/BeerGoggles.css

```diff
@@ -0,0 +1,54 @@
+.beerGooglesContainer {
+    max-width: 1200px;
+    margin: auto;
+}
+
+.beerGooglesIntroContainer {
+    display: flex;
+    padding-bottom: 30px;
+}
+
+.beerGooglesDescription {
+    font-size: 18px;
+    font-weight: 400;
+    padding: 0px 50px 0 25px;
+    width: 600px;
+}
+
+.beerGogglesEpHeader {
```

Comment:
I feel like this header's css styling is probably not unique to this page and can probably be a class in the main css file to be used on every project page?